src/deepsparse/debug_analysis.py

```diff
@@ -140,6 +140,29 @@ def parse_args():
         type=str,
         default="",
     )
+    parser.add_argument(
+        "--disable-batch-override",
+        help="Ignores the batch_size parameter",
+        action="store_true",
+        default=False,
+    )
+    parser.add_argument(
+        "--use-kvcache", help="Enable KVCache", action="store_true", default=False
+    )
+    parser.add_argument(
+        "--kv-cache-prev-num-tokens",
+        help="KVCache: The amount of previous tokens that will be read"
+        " from the external KV cache on the first inference",
+        type=int,
+        default=None,
+    )
+    parser.add_argument(
+        "--kv-cache-num-frozen-tokens",
+        help="KVCache: The amount of first tokens that we want to keep"
+        " permanently in the KV cache",
```

Comment:
can you clarify this comment?  does it mean the first N tokens will be frozen?

Reply:
yeah, exactly that. For some reason we need this for OPT models